src/main.js

```diff
@@ -135,14 +141,35 @@ function randomizePoster() {
   mainImage.src = images[getRandomIndex(images)];
   mainTitle.innerText = titles[getRandomIndex(titles)];
   mainQuote.innerText = quotes[getRandomIndex(quotes)];
+  currentPoster = new Poster(
+    mainImage.src,
+    mainTitle.innerText,
+    mainQuote.innerText
+  );
 }
 
 function toggleMakePoster() {
-  viewMainPoster.classList.toggle('hidden');
-  viewMakeOwnPoster.classList.toggle('hidden')
+  viewMainPoster.classList.toggle("hidden");
+  viewMakeOwnPoster.classList.toggle("hidden");
 }
 
 function toggleShowSavedPoster() {
-  viewMainPoster.classList.toggle('hidden');
-  viewShowSavedPosters.classList.toggle('hidden')
+  viewMainPoster.classList.toggle("hidden");
+  viewShowSavedPosters.classList.toggle("hidden");
+}
+
+function clickShowMyPoster() {
+  mainImage.src = inputImageURL.value;
+  mainTitle.innerText = inputPosterTitle.value;
+  mainQuote.innerText = inputPosterQuote.value;
+  currentPoster = new Poster(
+    mainImage.src,
+    mainTitle.innerText,
+    mainQuote.innerText
+  );
+  event.preventDefault();
```

Comment:
I like to see preventDefault happen at the beginning of the function. This improves readability when understanding what this method is doing.